frontend/passkeys.js

```diff
@@ -279,16 +312,15 @@ function getPasskeyNameFromUser() {
             e.preventDefault();
             const passkeyName = input.value.trim();
 
-            if (!passkeyName || !alphanumericRegex.test(passkeyName) || passkeyName.length > maxLength) {
-                // Show validation error
+            if (!passkeyName || !alphanumericRegex.test(passkeyName)) {
                 input.classList.add('invalid');
                 errorMessage.style.display = 'block';
-                
-                if (passkeyName.length > maxLength) {
-                    errorMessage.textContent = `Passkey name too long: maximum ${maxLength} characters allowed`;
-                } else {
-                    errorMessage.textContent = 'Cannot save passkey name: only letters, numbers, spaces, underscores and hyphens are allowed';
-                }
+                errorMessage.textContent = 'Cannot save passkey name: only letters, numbers, spaces, underscores and hyphens are allowed';
+                return;
+            } else if (passkeyName.length > maxLength) {
+                input.classList.add('invalid');
+                errorMessage.style.display = 'block';
+                errorMessage.textContent = `Passkey name too long: maximum ${maxLength} characters allowed`;
```

Comment:
This looks to be something else that's unrelated to the delete

Reply:
You're right, this was part of the name duplication validation that this branch was based on - now reverted